src/documentation/markdown/2020/lar-data-fields.md

```diff
@@ -400,7 +400,7 @@
   - 48%
   - 49%
   - 50%-60%
-  - >60%
+  - 60%
```

Comment:
Correct bin label?

Reply:
Has been changed back to >60% via standup feedback.